configs/atlas-xcache/config.d/40-atlas-xcache-proxy.cfg

```diff
@@ -0,0 +1,17 @@
+#
+# Configure the proxy plugin for the ATLAS XCache server.
+#
+# **********************************************************************
+# * WARNING: DO NOT EDIT THIS FILE.  IT WILL BE OVERWRITTEN ON UPGRADE *
+# **********************************************************************
+#
+# This file is part of the ATLAS XCache daemon
+
+# Clients populate this value so set it to a dummy value
+set proxyorigin==
```

Comment:
```suggestion
set proxyorigin=
```
no?

Reply:
Nope, because we use `pss.origin $(proxyorigin)` and ATLAS want to have the client decide the origin, e.g. [pss.origin =](http://xrootd.org/doc/dev49/pss_config.htm).

Reply:
no I just mean you had two "=" there... is that valid syntax?

Reply:
Yeah, because we need `proxyorigin`'s value to be `=` in the ATLAS case. I forget if I tested this but I hope it does. @abh3, should this work?

Reply:
Can you add a comment to that effect so I don't make a PR to fix it 3 months later? :-)